CODEOWNERS

```diff
@@ -104,3 +104,7 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/network/local_ratelimit @mattklein123 @junr03
 /*/extensions/filters/http/aws_request_signing @rgs1 @derekargueta @mattklein123 @marcomagdy
 /*/extensions/filters/http/aws_lambda @mattklein123 @marcomagdy @lavignes
+/*/extensions/filters/http/decompressor @junr03 @rojkov
```

Comment:
Should this be moved below `# Compression`?

Reply:
I guess the filter qualifies under the "compression" area. I originally meant it as a compression library area.